### PR TITLE
PlaceLab UI Tests Update

### PR DESCRIPTION
**Pull request by HarisJasarevic**

### CODE REVIEW

File: src/test/java/com/PlaceLab/HarisJasarevic/qamp/pages/LoginPage.java [hunk 1]
```diff
@@ -16,13 +16,15 @@ public class LoginPage {
     private final static String EXPECTED_ERROR_MSG = "Invalid credentials!";
 
     private final WebDriver driver;
+    private BasePage basePage;
```
Comment (nejlamaksumic-abh) [inline src/test/java/com/PlaceLab/HarisJasarevic/qamp/pages/LoginPage.java:19]: base page should be extended, and must not be instantiated in the page object classes, it should be abstract

File: src/test/java/com/PlaceLab/HarisJasarevic/qamp/pages/SinglePlaceReportPage.java [hunk 2]
```diff
@@ -27,64 +36,63 @@ public class SinglePlaceReportPage {
...
+        waitUntilClickable(CREATE_REPORT_BTN).click();
+        reportID = driver.getCurrentUrl().substring(driver.getCurrentUrl().lastIndexOf("/") + 1);
+        return reportID;
+}
+
+    public void deleteReport( final String reportID ) {
+        waitUntilClickable(getReportCheckbox(reportID)).click();
+        waitUntilClickable(DELETE_ICON).click();
+        waitUntilClickable(CONFIRM_DELETE_REPORT).click();
     }
```
Comment (nejlamaksumic-abh) [inline src/test/java/com/PlaceLab/HarisJasarevic/qamp/pages/SinglePlaceReportPage.java:87]: are you deleting the report from the SinglePlaceReportPage?

File: src/test/java/com/PlaceLab/HarisJasarevic/qamp/tests/LoginTestWithInvalidCredentials.java [hunk 3]
```diff
@@ -1,33 +1,29 @@
...
 
     @Parameters("browser")
-    @BeforeMethod
-    public void setup(@Optional("chrome") final String browser) {
-        driver = WebDriverSetup.getWebDriver(browser);
-        driver.get("https://demo.placelab.com/");
+    @BeforeMethod (alwaysRun = true)
+    public void initPages () {
         this.loginPage = new LoginPage(driver);
         driver.manage().window().maximize();
```
Comment (nejlamaksumic-abh) [inline src/test/java/com/PlaceLab/HarisJasarevic/qamp/tests/LoginTestWithInvalidCredentials.java:21]: can you place this line in the method in the BaseTest as well?

File: src/test/java/com/PlaceLab/HarisJasarevic/qamp/pages/LoginPage.java (hunk 1 above)
Comment (HarisJasarevic) [reply src/test/java/com/PlaceLab/HarisJasarevic/qamp/pages/LoginPage.java:19]: Thanks a lot Nejla for your time, I'll make the changes.

**Pull request merged**